docs/docs/sourcing-from-woocommerce.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Sourcing from WooCommerce
+---
+
+WooCommerce is the e-commerce platform for WordPress. This guide will serve as a helpful reference whether you’re interested in getting started with e-commerce for your WordPress site or beginning to use Gatsby with your current WooCommerce setup.
+
+## Prerequisites
+
+This guide assumes some familiarity with WordPress. You may want to read about [Sourcing from WordPress](/docs/sourcing-from-wordpress/) or step through a [tutorial on how to build a blog with WordPress and Gatsby](/blog/2019-04-26-how-to-build-a-blog-with-wordpress-and-gatsby-part-1) before diving in.
+
+You’ll need a WordPress site with the [WooCommerce](https://woocommerce.com/) plugin installed and activated.
+
+## Existing plugins
+
+This guide assumes the use of the [`gatsby-source-woocommerce` plugin](/packages/@pasdo501/gatsby-source-woocommerce/). It is a straightforward solution specific to WooCommerce. 
+
+Install it:
+
+```bash
+npm install --save @pasdo501/gatsby-source-woocommerce
+``` 
+
+Configure its options:
+
+```js:title=gatsby-config.js
+{       
+  resolve: "@pasdo501/gatsby-source-woocommerce",
+  options: {
+    // Base URL of Wordpress site
+    api: 'wordpress.domain',
+    // true if using https. false if nah.
```

Comment:
```suggestion
    // true if using https. false otherwise.
```

Reply:
Ha! I left that in there from the plugin but I'm with you here.